cpp/velox/compute/VeloxBackend.cc

```diff
@@ -56,7 +57,7 @@ VeloxBackend::VeloxBackend(const std::unordered_map<std::string, std::string>& c
   // mem tracker
   int64_t maxMemory;
   {
-    auto got = confMap_.find(kSparkTaskOffHeapMemory); // per task, for creating iterator
+    auto got = confMap_.find(kSparkOffHeapMemory); // per executor, shared by tasks for creating iterator
```

Comment:
Question: [```memPoolOptions_```](https://github.com/oap-project/gluten/blob/5c7d94a3866c8f927ba114af8400f0f04022ff45/cpp/velox/memory/VeloxMemoryPool.cc#L426) will be used when calling [```GetResultIterator```](https://github.com/oap-project/gluten/blob/5c7d94a3866c8f927ba114af8400f0f04022ff45/cpp/velox/compute/VeloxBackend.cc#LL95C47-L95C64) by each task. How is it shared by tasks?

Reply:
Previously we passed the option to each task but in the PR I have changed to this way:
https://github.com/zhztheplayer/gluten/blob/4e8b486cf72546e5b414c560bf227a286990fcf7/cpp/velox/memory/VeloxMemoryPool.cc#L455-L465

The process-wise static pool will be the parent pool of all context pools for the tasks, so the cap will be shared by the tasks.